src/MSALWrapper.Test/AuthFlowTest/AuthFlowBrokerTest.cs

```diff
@@ -396,6 +398,27 @@ public async Task BrokerAuthFlow_GetTokenInteractive_OperationCanceledException_
             authFlowBroker.ErrorsList[2].Message.Should().Be("Interactive Auth (with extra claims) timed out after 15 minutes.");
         }
 
+        /// <summary>
+        /// Ensure <see cref="IPCAWrapper.WithPromptHint"/> be invoked in <see cref="TokenFetcherPublicClient.GetTokenNormalFlowAsync"/>.

```

Comment:
Are these references still valid? 🤔

Reply:
This is a splendid catch. Fixed it!

Reply:
@shalinikhare27: Isn't `IPCAWrapper` also invalid? I thought we did away with that interface... 🤔

Reply:
Oh, nvm, it appears to still be there.